### PR TITLE
CRAB: Enable TTS in Crab Engine

### PR DESCRIPTION
**Pull request by Brakkugit**

### CODE REVIEW

File: engines/crab/ui/ChapterIntro.cpp [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 #include "crab/animation/sprite.h"
 #include "crab/ui/ChapterIntro.h"
+#include "common/text-to-speech.h"
```
Comment (sev-) [inline engines/crab/ui/ChapterIntro.cpp:33]: why is it here?

File: engines/crab/ui/ChapterIntro.h [hunk 2]
```diff
@@ -66,6 +67,7 @@ class ChapterIntro {
 
 	void draw(pyrodactyl::event::Info &info, Common::String &text,
 			  pyrodactyl::anim::Sprite *curSp, const pyrodactyl::people::PersonState &state);
+
```
Comment (sev-) [inline engines/crab/ui/ChapterIntro.h:70]: please revert this line change

**Review by sev- — CHANGES_REQUESTED** [review]
added two minor notes

**Comment by sluicebox** [comment]
Also: this doesn't build! =)

See the red X for details

**Pull request merged**